src/postgraphile/http/subscriptions.ts

```diff
@@ -65,8 +66,9 @@ export async function enhanceHttpServerWithSubscriptions<
     handleErrors,
   } = postgraphileMiddleware;
   const pluginHook = pluginHookFromOptions(options);
-  const externalUrlBase = options.externalUrlBase || '';
-  const graphqlRoute = options.graphqlRoute || '/graphql';
+  const graphqlRoute =
+    subscriptionServerOptions?.graphqlRoute ||
```

Comment:
This optional chaining syntax does not work for me in the Docker build. I get: 
```
/postgraphile/build-turbo/postgraphile/http/subscriptions.js:40                                                           
    const graphqlRoute = subscriptionServerOptions?.graphqlRoute ||                                                       
                                                   ^
SyntaxError: Unexpected token '.'

```

Reply:
Thanks for raising this! Fixed in https://github.com/graphile/postgraphile/pull/1376

In future, please file an issue rather than commenting on a closed PR; it makes it easier to track :pray: